config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -134,7 +134,7 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --check-leaked-resources
-      - --env=KUBE_GCE_MASTER_IMAGE=cos-93-16623-341-29
+      - --env=KUBE_GCE_MASTER_IMAGE=cos-97-16919-235-48
```

Comment:
any reason for updating cos image? are we using cos97 at all other places?

Reply:
it was updated to cos-93 a few weeks back for fixing a bash version issue. Since cos97 shipped with cgroupv2, thought we should update this also to cos97.